addons/a11y/.storybook/components/Button/stories.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { checkA11y } from './../../../src';
+
+import Button from './component';
+
+import Faker from 'faker';
+
+const text = Faker.lorem.words();
+
+storiesOf('<Button />', module)
+  .addDecorator(checkA11y)
```

Comment:
I think these a11y stories should be moved over to CRA so we can test them together.

Reply:
agree

Reply:
Added, will leave the old storybook, because might copy over some more stories later.